src/fidesops/task/graph_task.py

```diff
@@ -152,11 +160,7 @@ def to_dask_input_data(self, *data: List[Row]) -> Dict[str, List[Any]]:
 
             for row in rowset:
                 for foreign_field_path, local_field_path in field_mappings:
-                    new_value = (
-                        row[foreign_field_path.string_path]
-                        if foreign_field_path.string_path in row
-                        else None
-                    )
+                    new_value = foreign_field_path.retrieve_from(row)
```

Comment:
Data is returned from mongo in a potentially nested format.  This fix lets us retrieve potentially nested data before passing as flattened format into a downstream collection that needs potentially nested data as input.

Docstring updated.